pokemongo_bot/__init__.py

```diff
@@ -62,7 +62,7 @@ def update_web_location(self, cells=[], lat=None, lng=None, alt=None):
             status = map_objects.get('status', None)
             cells = map_objects['map_cells']
 
-            #insert detail info about gym to fort
+            # insert detail info about gym to fort
```

Comment:
It's unfortunate that you have so many changes in this file. It makes it very difficult to read the diff here and likely that it will cause someone else merge conflicts.